### PR TITLE
Upgrade embeded setuptools

### PR DESCRIPTION
**Pull request by gaborbernat**

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -61,7 +61,7 @@ virtualenv.activate =
     batch = virtualenv.activation.batch:BatchActivator
     fish = virtualenv.activation.fish:FishActivator
     powershell = virtualenv.activation.powershell:PowerShellActivator
-    python = virtualenv.activation.python:PythonActivator
```
Comment (RowdyHowell) [inline setup.cfg:64]: I'm getting an error that might be related to this merge.  Is the change expected?

`AttributeError: module 'virtualenv.activation.python' has no attribute 'PestythonActivator'`

**Comment by victor-torres** [comment]
Same issue here:

```
AttributeError: module 'virtualenv.activation.python' has no attribute 'PestythonActivator'
```

File: setup.cfg (hunk 1 above)
Comment (brutimus) [reply setup.cfg:64]: This just broke all of my production build pipelines as well.

Comment (gaborbernat) [reply setup.cfg:64]: I have to be honest, have no idea how this change made it in. Late Friday releases are always dangerous but wanted to get out a fix release to address peoples issues due to https://github.com/pypa/virtualenv/pull/1846 🤦 🤷‍♂️

Comment (victor-torres) [reply setup.cfg:64]: It happens. Thanks for the quick fix! @gaborbernat

**Comment by phrfpeixoto** [comment]
Haha. It seems we're missing tests here.

**Pull request merged**